.gitignore

```diff
@@ -3,3 +3,4 @@ foundryconfig.json
 dist
 .DS_Store
 .env
+mprocs.yaml
```

Comment:
I figured, while I'm in here, I might as well ignore the config script for the tool I use to run dev and Foundry from one terminal window.